src/components/schemas/expense_policies.yaml

```diff
@@ -0,0 +1,719 @@
+      example: 10000
+    expense_amount_min:
+      allOf:
+        - $ref: ./fields.yaml#/amount
+      description: |
+        Minimum amount below which policy will be violated.
+      example: 500
+    expense_distance_max:
+      allOf:
+        - $ref: ./fields.yaml#/distance
+      description: |
+        Maximum travel distance allowed, above which policy will be violated.
+      example: 10000
+    expense_distance_min:
+      allOf:
+        - $ref: ./fields.yaml#/distance
+      description: |
+        Minimum travel distance allowed, below which policy will be violated.
+      example: 500
+    expense_category_ids:
+      type: array
+      items:
+        $ref: './fields.yaml#/id_integer'
+      description: |
+          List of category ids targeted by this policy.
+      example: [2222, 1221]
+    expense_category_ids_op:
+      $ref: ./fields.yaml#/in_operator
+    expense_project_ids:
+      type: array
+      items:
+        $ref: './fields.yaml#/id_integer'
+      description: |
+        List of project ids targeted by this policy.
+      example: [1234, 5678]
+    expense_project_ids_op:
+      $ref: ./fields.yaml#/in_operator
+    expense_cost_center_ids:
+      type: array
+      items:
+        $ref: './fields.yaml#/id_integer'
+      description: |
+        List of cost centers ids targeted by this policy.
+      example: [1987, 3456]
+    expense_cost_center_ids_op:
+      $ref: ./fields.yaml#/in_operator
+    expense_allowed_exchange_rate_difference_percentage:
+      type: number
+      description: |
+        Allowed percentage difference for exchange rates above which policy will be violated.
+      example: 10
+      minimum: 0
+      maximum: 100
+    expense_allowed_distance_difference_percentage:
+      type: number
+      description: |
+        Allowed percentage difference for travel distance above which policy will be violated.
+      example: 8
+      minimum: 0
+      maximum: 100
+    expense_custom_fields:
+      allOf:
+        - $ref: './fields.yaml#/custom_fields'
+      description: |
+        Custom fields
+    expense_custom_fields_op:
+      type: object
+      description: |
+        Operator for the expense_custom_fields.
+      properties:
+        field_name:
+          type: string
+    expense_locations:
+      description: |
+        This field will be present for travel related categories.
+        For travel between Place A to Place B, this array will contain two items, From location at array index 0 and To location at array index 1.
+      type: array
+      items:
+        $ref: './fields.yaml#/location'
+    expense_locations_op:
+      $ref: ./fields.yaml#/in_operator
+    expense_creation_delay_days_max:
+      type: number
+      example: 12
+      description: |
+        No of days from spent date after which if expense is created will violate policy.
+    expense_on_holiday_allowed:
+      type: boolean
+      description: |
+        Is expense on holiday allowed.
```

Comment:
If this is set to false, then for all such expenses which/that are created on holidays, this policy rule will be violated.